portal-web/test/functional/com/liferay/portalweb/tests/MyBlogTests/MyBlogTests.testcase

```diff
@@ -0,0 +1,54 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given the Blogs Portlet is added to the page") {
```

Comment:
I **really** liked that you added the given already in the setup part, but so that we don't have any steps established outside of the Given-When-Then pattern, move these steps that are outside into the task to inside, since they are part of the Given process